src/e2e-test/java/io/cdap/plugin/bigquery/stepsdesign/BigQueryBase.java

```diff
@@ -110,13 +119,13 @@ public void validateRecordsTransferredToTargetTableIsEqualToNumberOfRecordsFromS
 
   @Then("Validate partition date in output partitioned table")
   public void validatePartitionDateInOutputPartitionedTable()
-    throws IOException, InterruptedException {
+          throws IOException, InterruptedException {
     Optional<String> result = BigQueryClient
-      .getSoleQueryResult("SELECT distinct  _PARTITIONDATE as pt FROM `" +
-                            (PluginPropertyUtils.pluginProp("projectId")) + "." +
-                            (PluginPropertyUtils.pluginProp("dataset")) + "." +
-                            TestSetupHooks.bqTargetTable +
-                            "` WHERE _PARTITION_LOAD_TIME IS Not NULL ORDER BY _PARTITIONDATE DESC ");
+            .getSoleQueryResult("SELECT distinct  _PARTITIONDATE as pt FROM `" +
```

Comment:
Why are these changes are made, revert all the unnecessary changes made.

Reply:
I didn't touch this line.